export const err404 = {
  init: async () => {
    console.log('404')
  }
}
